lib/bandit/http1/handler.ex

```diff
@@ -88,6 +100,50 @@ defmodule Bandit.HTTP1.Handler do
     end
   end
 
+  defp maybe_upgrade_h2c(req, transport_info, method, request_target, headers) do
+    with {:upgrade, "h2c"} <- {:upgrade, Bandit.Headers.get_header(headers, "upgrade")},
+         %Bandit.TransportInfo{secure?: false} <- transport_info,
+         {:ok, connection_headers} <- get_connection_header_keys(headers),
+         {:settings, [{"http2-settings", settings_payload}]} <-
+           {:settings, Enum.filter(headers, fn {key, _value} -> key == "http2-settings" end)},
+         {:ok, settings_payload} <- Base.url_decode64(settings_payload) do
+      Bandit.HTTP1.Adapter.send_resp(
```

Comment:
Does this not format onto a single line?

Reply:
I have update this (send_resp) in ee4cdfb. Let me know what you think.